src/plugins/unified_search/public/typeahead/_index.scss

```diff
@@ -1 +1,164 @@
-@import 'suggestion';
+// These are the various types in the dropdown, they each get a color
+$kbnTypeaheadTypes: (
```

Comment:
Nit, since this file contains styles and is not just a manifest it should be named not as `index`. Can you rename back to `_suggestion.scss`, then in the `index` file that does import this one, you can just import `_suggestion` directly.

Reply:
Done https://github.com/elastic/kibana/pull/131784/commits/89673f62f065090abf236e5796316d335a98ec0d